src/styles/grid/src/_macro.njk

```diff
@@ -0,0 +1,3 @@
+{% macro onsGrid(params) %}
```

Comment:
Only comment for me is I'd say we should name this `patternLibExampleGrid` to prevent any confusion that this macro should be used in production code.

Reply:
Yep good point.